src/main/java/org/folio/circulation/resources/CheckOutByBarcodeResource.java

```diff
@@ -41,10 +45,14 @@
 import org.folio.circulation.services.EventPublisher;
 import org.folio.circulation.support.Clients;
 import org.folio.circulation.support.RouteRegistration;
+import org.folio.circulation.support.ValidationErrorFailure;
 import org.folio.circulation.support.http.OkapiPermissions;
 import org.folio.circulation.support.http.server.HttpResponse;
+import org.folio.circulation.support.http.server.ValidationError;
 import org.folio.circulation.support.http.server.WebContext;
+import org.folio.circulation.support.results.FailedResult;
 import org.folio.circulation.support.results.Result;
+import org.folio.circulation.support.results.SuccessfulResult;
```

Comment:
Why do we have the changes in import if nothing has been changed in the class?

Reply:
fixed